tokio-tar/README.md

```diff
@@ -83,10 +83,10 @@ fn main() {
 
 This project is licensed under either of
 
- * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
-   http://www.apache.org/licenses/LICENSE-2.0)
- * MIT license ([LICENSE-MIT](LICENSE-MIT) or
-   http://opensource.org/licenses/MIT)
+- Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
```

Comment:
❓ Is `*` not valid Markdown syntx?

Reply:
It is, the formatter just doesn't like it apparently :D